src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -128,6 +131,12 @@ export default {
 
 			return displayName
 		},
+
+		disableMenu() {
+			// disable the menu if accessing the conversation as guest
+			// or the message sender is a bridged user
+			return this.$store.getters.getActorType() === 'guests' || this.actorType === ATTENDEE.ACTOR_TYPE.BRIDGED
```

Comment:
Should be disabled unless the actor type is user?

```suggestion
			return this.$store.getters.getActorType() === 'guests' || this.actorType === ATTENDEE.ACTOR_TYPE.USERS
```

Or do bots etc not fall into the path?

Reply:
Logic follows the same as we had on removed component
Code will be captured before it will reach NcAvatar for: guests, deleted users, bots, changelog, emails